index.js

```diff
@@ -2179,6 +2184,13 @@
     };
   }
 
+  //  see :: (String, { name :: String, url :: String }) -> String
+  function see(label, record) {
+    return record.url &&
```

Comment:
If `record.url` is undefined, this function would return `undefined`, right, violating the type signature? I'm looking into it because of fluture-js/fluture-sanctuary-types#5. I believe if sanctuary-def receives type classes from an older version of sanctuary-type-classes, its errors will now include: `"\n undefined"`.

Alternatively, the type class `@@type` property has changed (`@version`-part?) and sanctuary-def will throw an error if getting an older type class ... in which case older versions of sanctuary-type-classes no longer work with newer versions of sanctuary-def, making https://github.com/fluture-js/Fluture/commit/37822c5068f3c207212fd934735e67af40cf19f3 an indirect breaking change, forcing me to bump the version of fluture-sanctuary-types.